docscan/session/create/sdk_config_test.go

```diff
@@ -0,0 +1,28 @@
+package create
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+func ExampleSdkConfigBuilder_Build() {
+	notifications, err := NewSdkConfigBuilder().
+		WithAllowsCamera().
+		WithErrorUrl("https://errorurl.com").
+		WithFontColour("#ff0000").
+		WithLocale("fr_FR").
+		WithPresetIssuingCountry("USA").
+		WithPrimaryColour("#aa1111").
+		WithSecondaryColour("#bb2222").
+		WithSuccessUrl("https://successurl.com").
+		Build()
+
+	if err != nil {
+		fmt.Printf("error: %s", err.Error())
+		return
+	}
+
+	data, _ := json.Marshal(notifications)
+	fmt.Println(string(data))
+	// Output: {"allowed_capture_methods":"CAMERA","primary_colour":"#aa1111","secondary_colour":"#bb2222","font_colour":"#ff0000","locale":"fr_FR","preset_issuing_country":"USA","success_url":"https://successurl.com"}
```

Comment:
`error_url` should be present in this JSON - `WithErrorUrl()` is currently setting the success URL

Reply:
Good catch, fixed in fc4fef1